pkg/kine/server/update.go

```diff
@@ -46,26 +46,28 @@ func (l *LimitedServer) update(ctx context.Context, rev int64, key string, value
 
 	if rev == 0 {
 		rev, err = l.backend.Create(ctx, key, value, lease)
-		ok = true
-
-		span.SetAttributes(
-			attribute.Int64("revision", rev),
-			attribute.Bool("ok", ok),
-		)
+		if err == ErrKeyExists {
```

Comment:
let's use `errors.Is()`, because: https://go.dev/play/p/3Uo9WyhnQeH

Reply:
While I in general agree with wrapping an error, I would disagree with this one alone. We would need that only if wrapping and in this case, I think we shouldn't.

This kind of errors are part of the contract between the abstraction and part of the flow: wrapping it in a user-friendly one would only make you allocate to discard it immediately when you are handling it. Then why doing it?